var/spack/repos/builtin/packages/htslib/package.py

```diff
@@ -76,13 +81,23 @@ def url_for_version(self, version):
             url = "https://github.com/samtools/htslib/releases/download/{0}/htslib-{0}.tar.bz2"
             return url.format(version.dotted)
 
+    def flag_handler(self, name, flags):
+        if name == "cflags" and self.spec.satisfies("+pic"):
+            flags.append("-fPIC")
+        return (flags, None, None)
+
     def configure_args(self):
         spec = self.spec
         args = []
 
         if spec.satisfies("@1.3:"):
             args.extend(self.enable_or_disable("libcurl"))
 
+        if spec.satisfies("@1.5:"):
```

Comment:
This check doesn't match the variant definition, so it might be you forgot a `:` when defining the variant, i.e. `@1.5:` instead of `@1.5`.

Reply:
Also, is it true that versions from 1.3 will provide `s3` and `gcs` support, as stated in the variant description for `libcurl`?

Reply:
From 1.5 on these were configurable. Prior to that they were always built.